.travis/stp.sh

```diff
@@ -33,6 +33,23 @@ if [ "x${STP_VERSION}" != "x" ]; then
   sudo make install
   cd ../../
 
+  # Determine STP build flags
```

Comment:
Oh I see. I'll fix that now. We might not be able to maintain this though. There is not exact correspondence between these options and this may get worse over time.